common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -70,100 +73,93 @@ public static String modifyMultipleMatchKeyword(String query){
     public static final String STANDARD_CONCEPT_CODE = "S";
     public static final String CLASSIFICATION_CONCEPT_CODE = "C";
 
-    public Slice<Concept> searchConcepts(String query,
-                                         StandardConceptFilter standardConceptFilter, List<String> vocabularyIds,
-                                         List<String> domainIds, int limit, int minCount) {
+    public Slice<Concept> searchConcepts(String query, StandardConceptFilter standardConceptFilter, List<String> vocabularyIds, List<String> domainIds, int limit, int minCount) {
 
 
         Specification<Concept> conceptSpecification =
                 (root, criteriaQuery, criteriaBuilder) -> {
                     List<Predicate> predicates = new ArrayList<>();
+                    List<Predicate> standardConceptPredicates = new ArrayList<>();
+                    standardConceptPredicates.add(criteriaBuilder.equal(root.get("standardConcept"),
+                            criteriaBuilder.literal(STANDARD_CONCEPT_CODE)));
+                    standardConceptPredicates.add(criteriaBuilder.equal(root.get("standardConcept"),
+                            criteriaBuilder.literal(CLASSIFICATION_CONCEPT_CODE)));
+
+                    List<Predicate> nonStandardConceptPredicates = new ArrayList<>();
```

Comment:
standardConceptPredicates and nonStandardConceptPredicates look identical... is that what you intended?

Reply:
Sorry, missed the "not" part. :)